src/main/java/edu/ucsb/cs156/happiercows/entities/Commons.java

```diff
@@ -29,6 +29,7 @@
   private double milkPrice;
   private double startingBalance;
   private LocalDateTime startingDate;
+  private double degradationRate;
```

Comment:
Please consider adding this field to the fixtures for the frontend as well.

That would be done in the files: 

https://github.com/ucsb-cs156-s22/s22-6pm-happycows/blob/main/frontend/src/fixtures/commonsFixtures.js

https://github.com/ucsb-cs156-s22/s22-6pm-happycows/blob/main/frontend/src/fixtures/userCommonsFixtures.js

Reply:
Coordinated with Zack and he will be adding the showLeaderboard and degradationRate to the fixtures files so that all fixtures changes will be in PR #40